### PR TITLE
[JAVA-8364] Split spring-5-reactive-2 module

### PR DESCRIPTION
**Pull request by hkhan**

### CODE REVIEW

File: spring-5-reactive-3/pom.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <dependency>
+            <groupId>io.projectreactor</groupId>
+            <artifactId>reactor-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>integration-lite-first</id>
```
Comment (kwoyke) [inline spring-5-reactive-3/pom.xml:43]: Do we really need this extra config? Shouldn't Spring Boot pick up logback-test.xml "out of the box"?

File: spring-5-reactive-3/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+	<include
+		resource="org/springframework/boot/logging/logback/base.xml" />
+	<appender name="LISTAPPENDER"
```
Comment (kwoyke) [inline spring-5-reactive-3/src/test/resources/logback-test.xml:5]: There's no ListAppender in this module.

Comment (hkhan) [reply spring-5-reactive-3/src/test/resources/logback-test.xml]: removed

File: spring-5-reactive-3/pom.xml (hunk 1 above)
Comment (hkhan) [reply spring-5-reactive-3/pom.xml]: yes, you're right, logback-test.xml is used by default - fixed.

**Comment by kwoyke** [comment]
retest this please

**Pull request merged**